demo/petstore/openapi.json

```diff
@@ -398,7 +398,7 @@
         ],
         "summary": "Deletes a pet",
         "description": "",
-        "operationId": "deletePet",
+        "operationId": "https://auth.example.org/operations/petstore/delete-pet-by-id",
```

Comment:
Think this can just be `delete-pet`, minor gripe though

Reply:
mostly just coping the same schema used for get-pet